### ISSUE TITLE
Can't launch to moon

### ISSUE THREAD

**Issue by scorpius80** [micdoodle8/Galacticraft#2191]
Built a ship, can get to the planet selection screen.. select moon, press launch... and nothing... just stays in the planet screen.  Can return to overworld, but that's it.  Just created with the latest copy of galacticraft, so I am confused.  Ideas?

**Comment by EzerArch** [comment]
Right before the launch, does the countdown keep resetting/lagging?

**Comment by scorpius80** [comment]
Nope.  I launch fine, get to the planet selection screen, but the launch button does nothing with the moon selected

**Comment by Adkkid** [comment]
I'm having a similar problem, with all planets. Im not sure if its the same as you, but the launch button is red, and the tier requirement is also red. However, this is only happening on a server. The same modpack in single player works just fine. Is anyone aware of an issue with Thermos 55, Multiverse 2.5, and Galacticraft version 413-454 causing this? Ive been tearing my hair out trying to find the problem.

**Comment by BlesseNtumble** [comment]
@Adkkid need sync configs. Client configs = Server configs.

**Comment by Adkkid** [comment]
Oh, whoops. Forgot to update this when I figured it out. I spent a few hours trying some different things, and as I thought, it was a plugin problem. It looks like by having the dimensions set to auto load and keep the spawn region in memory via Multiverse AND using the static loading from Galacticraft, it prevented our Cauldron.yml from forcing the world to stay loaded.

I probably should have figured it out sooner, but it looks like it was just a config problem on our end and not an actual bug. Thanks, though, @BlesseNtumble
